.env

```
@@ -0,0 +1,16 @@
+NODE_ENV=development
+PORT=3000
+JWT_SECRET=test-secret-key-for-development
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the NODE_ENV key

(UnorderedKey)